File: hoosong0235/Bronze/5086. 배수와 약수/배수와 약수.py
while True:
    nums = list(map(int, input().split()))
    if nums == [0, 0]:
        break
    elif nums[1] % nums[0] == 0:
        print("factor")
    elif nums[0] % nums[1] == 0:
        print("multiple")
    else:
        print("neither")

File: hoosong0235c/Bronze/5086. 배수와 약수/배수와 약수.c
#include <stdio.h>

int main()
{
    int n, m;

    while (1)
    {
        scanf("%i %i", &n, &m);

        if (!n && !m)
            break;

        if (n % m == 0)
            printf("multiple\n");
        else if (m % n == 0)
            printf("factor\n");
        else
            printf("neither\n");
    }
}